app/utils/text_processing.py

```
@@ -131,94 +137,419 @@ def remove_stopwords(tokens: list, language:str ='en'):
+        max_keywords (int): 최대 추출할 키워드 수 (기본값: 20)
+    
+    Returns:
+        List[str]: 추출된 키워드 리스트
+    """
+    if not text:
+        return []
+    
+    # 불용어 로드
+    try:
+        korean_stopwords = set(load_korean_stopwords())
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)